packages/aurora-file-io/index.js

```diff
@@ -10,23 +10,34 @@ function save(editorState) {
   const context = getAuroraDirContext();
   const note = attachMetaData(editorState);
   context.writeAsync(noteFileName(note), note);
+  return note["date"];
+}
+
+function deleteNote(uuid) {
+  const context = getAuroraDirContext();
+  context.removeAsync(uuid + noteFileExt);
 }
 
 function loadNotes(callback) {
   const context = getAuroraDirContext();
   const noteFiles = context.find({ matching: "*" + noteFileExt });
   const notes = [];
+
   noteFiles.forEach(file => {
     context.readAsync(file).then(data => {
-      const contentState = convertFromRaw(JSON.parse(data).contentState);
+      var json = JSON.parse(data);
+      const contentState = convertFromRaw(json.contentState);
       const editorState = EditorState.createWithContent(contentState);
-      notes.push(editorState); // Just return editor state for now
-      if (notes.length === noteFiles.length) {
-        // this means all notes have been loaded
+      var note = {
+        "uuid":json["date"],
```

Comment:
You may want to change the `uuid` name to be "date" or something so we don't run into confusion with naming. Or maybe just `id` so if we can change this in the future we're not causing problems. 

Also:
``` js 
json.date
```